provisioning/device/src/Utilities/X509CertificateStore.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Utilities
+{
+    internal class X509CertificateStore : ICertificiateStore, IDisposable
```

Comment:
Could use doc comments to explain what this class is and why it exists, like, to enable mocking of `CertfiicateInstaller` so unit tests don't need to mess with the system cert store.